### DISCUSSION TITLE
Use prefixed env variables in sdk

### DISCUSSION THREAD

**Discussion by aharelick** [vercel/community#2288] [Help] [answered]
### Summary

The @vercel/kv SDK wants to use KV_REST_API_URL and KV_REST_API_TOKEN by default. However, when I set up my kv storage I changed the prefix in the below picture to specify another name (not in the picture). How do I configure the sdk to use my prefix or new env variables?
<img width="476" alt="Screen Shot 2023-05-03 at 2 34 23 PM" src="https://user-images.githubusercontent.com/3924168/236054909-bf8d4f31-6bb0-4e55-960a-c3639dc02916.png">

**Comment by amyegan** [comment, marked as answer]
Hi @aharelick! You can use `createClient` to use custom environment variables https://github.com/vercel/storage/blob/main/packages/kv/README.md#custom-environment-variables
